src/styles/variables.js

```diff
@@ -92,3 +92,4 @@ export const overlayZIndex = '190'; // overlay-z-index
 export const tooltipZIndex = '190'; // tooltip-z-index
 export const bdlLineHeight = '20px'; // bdl-line-height
 export const bdlBorderRadiusSize = '4px'; // bdl-border-radius-size
+export const bdlTransitionDuration200Ms = '0.2s'; // bdl-transition-duration-200ms
```

Comment:
Do we want to go with @booc0mtaco's suggestion of `bdlTransitionDuration-short`?

Reply:
Another PR for that, this is auto-generated from prior PR.